src/pages/CompleteRegistration/CompleteRegistrationStyles.tsx

```diff
@@ -0,0 +1,106 @@
+import { Radio } from "antd";
+import Title from "antd/lib/typography/Title";
+import styled from "styled-components";
+import {
+  BLUE,
+  DARK_BLUE,
+  // GOOGLE_BTN_BACKGROUND,
```

Comment:
delete if not used

Reply:
Will do

Reply:
Done, I'll merge it with the #12  that still has pending things to do